### DISCUSSION TITLE
How can I get a repo indexed?

### DISCUSSION THREAD

**Discussion by Liturgist** [community/community#52095] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

GitHub Search (beta) reports that the target repo has not been indexed.

Is there anything I can do to cause an index to be done?

**Comment by meppu** [comment, marked as answer]
You just need to wait until it is indexed. Time depends on environments size.

**Reply by Liturgist** [reply]
> You just need to wait until it is indexed. Time depends on environments size.

I will mark this as the answer because I fear that it is correct. I am hoping for better.

**Reply by SildrePood** [reply]
How long do i have to wait?

**Comment by look** [comment]
@Liturgist we're working on better messaging, but sometimes when this message doesn't go away it means there is a problem indexing the repository. If you can give me more details, I can look into that for you.

**Reply by sysrage** [reply]
@look sorry for reviving a super-old thread, but I have a repo that's stuck indexing and the code hasn't changed in over a year. Is there some way to force indexing to restart?

Edit to avoid another unnecessary ping: it turns out I wasn't waiting long enough. Apparently just trying to search the repo triggered an indexing. It now works. Thanks!

**Reply by look** [reply]
@sysrage no problem, please file a [support request](https://support.github.com/) and we will look into it.

**Comment by dankoman39** [comment]
i can't prove it, but i'm pretty sure that attempting to search within an unindexed repo triggers its addition to an indexing queue.  would be nice if there were a flag or something you could set to prioritize indexing.

it makes sense why they do it this way though - since the index for a repository is for searchability purposes, there's really no point in indexing the data (from github's perspective) until they're sure it's a repo that is going to be searched.
